### PR TITLE
Fixed PEP8 coding style issues.

### PR DESCRIPTION
**Pull request by ypid**

### CODE REVIEW

File: yaml2rst.py [hunk 1]
```diff
@@ -64,10 +65,13 @@ def __init__(self, **keywordargs):
     pats['enum'] = ('(%(arabic)s|%(loweralpha)s|%(upperalpha)s|%(lowerroman)s'
                     '|%(upperroman)s|#)' % enum.sequencepats)
 
-    for format in enum.formats:
-        pats[format] = '(?P<%s>%s%s%s)' % (
-              format, re.escape(enum.formatinfo[format].prefix),
-              pats['enum'], re.escape(enum.formatinfo[format].suffix))
+    for format_key in enum.formats:
+        pats[format_key] = '(?P<{}>{}{}{})'.format(
```
Comment (htgoebel) [inline yaml2rst.py]: This is not PEP8 and it's less readable.

**Comment by htgoebel** [comment]
I think, this is a useless change, given the small number of changes and that most of them are whitespace only. Anyway I'm going to accept it (except of the `format`) (Other projects do not even accept whitespace-only changes, even if the code is ugly as can be.) But I dislike the change of `%` to `...format`, which is neither backed by PEP 8 nor is `%` deprecated. IMO the `format` syntax is less readable, see line comments.

**Comment by ypid** [comment]
Thanks for your feedback.

Can you maybe explain why you don’t like the `.format`? I don’t really think it
is less readable and `.format` has certain advantages. I have been using it
ever since. I don’t like the extra operator. `.format` feels more natural (when
coming from an OO background).

https://stackoverflow.com/questions/5082452/python-string-formatting-vs-format

**Comment by ypid** [comment]
The `%` is not deprecated as long as Python 2 is not considered to be deprecated. `.format` is the default string formating in Python 3. https://docs.python.org/3/library/string.html#string-formatting

**Comment by htgoebel** [comment]
The `.format` method is important and helpful when using localized strings. But for only concatenating stings it of not much use. It just adds characters to the source code. Which one to use is a matter of personal preferences here. I do not see _any_ benefit in change this.

Both your commit and pull-request say: "Fixed PEP8 coding style issues", which is plain wrong for `.format`.

Sidenote: While performance is not an issue for this program: In bytecode , `,format` is a bit slower since the attribute needs to be resolved and a call has to be made (which includes setting up a stack, etc.). In contrast `%` is a single opcode which is directly executed.

> The % is not deprecated as long as Python 2 is not considered to be deprecated.

`%` is not depricated even in Python 3.5, see https://docs.python.org/3/library/stdtypes.html#old-string-formatting

**Comment by ypid** [comment]
All right, I reverted my `.format` usage. You are totally right. It is just preference and it is not worth it having much of an argument about that. No offense. Thanks for considering the rest of the patch!

**Comment by htgoebel** [comment]
Thanks for taking your time :-)

**Pull request merged**